test/test-bopomofo.c

```diff
@@ -1690,6 +1691,49 @@ void test_KB()
     test_KB_MPS2();
 }
 
+void test_chewing_phone_to_bopomofo()
+{
+    char *u8phone;
+    char *rt;
+    int  len;
+    uint16_t phone;
+    uint16_t expect;
+
+    start_testcase(NULL, fd);
+
+    u8phone = "\xE3\x84\x86\xE3\x84\xA3" /* ㄆㄣ */ ;
+    phone = UintFromPhone(u8phone);
+    expect = 1104;
```

Comment:
I am afraid that it is a bit hard to maintain the test case. Do you have more flexible way to generate expected results?